packages/pigment-react/tests/testUtils.ts

```diff
@@ -32,8 +32,11 @@ export async function runTransformation(
       configFile: false,
       babelrc: false,
     },
-    tagResolver(_source: string, tag: string) {
-      return require.resolve(`../exports/${tag}`);
+    tagResolver(source: string, tag: string) {
+      if (source === '@pigment-css/react') {
+        return require.resolve(`../exports/${tag}`);
+      }
+      return null;
```

Comment:
Need to scope to `@pigment-css/react`, otherwise it will throw an error `"Cannot find '../exports/touchRippleClasses'"` from the input below.

```js
import { touchRippleClasses } from '@mui/material/ButtonBase';
```